### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by matrix880**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,8 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+
+        return Integer.toBinaryString(value);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BinaryString.java]: create your own implementation

Comment (matrix880) [reply src/main/java/core/basesyntax/BinaryString.java]: done

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,11 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder s = new StringBuilder();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BinaryString.java]: s - not informative name

Comment (matrix880) [reply src/main/java/core/basesyntax/BinaryString.java]: done

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**